diesel/src/sqlite/types/date_and_time/chrono.rs

```diff
@@ -636,22 +704,6 @@ mod tests {
             "1970-01-01T00:00:00+00:00",
             "1970-01-01T00:00:00.000+00:00",
             "1970-01-01T00:00:00.000000+00:00",
-            "1970-01-01 00:00+01:00",
```

Comment:
I assume these are removed because the serialization/deserialization of these values changed? Can you explain how and why they changed?

(We need to be very careful here to not introduce a breaking change for existing users)

Reply:
Agreed we need to be super careful, I consider this test to be a bug but it might be considered a breaking change depending on how you look at it.

Basically right now if you insert manually a timestamp with a timezone in sqlite and **read** it using diesel it will ignore the timestamp. So for example `1970-01-01T00:00:00-01:00` will be treated as if it is `1970-01-01T00:00:00Z` when it is in fact ` 1970-01-01T00:01:00Z`. So for me this is a bug, but it does mean that people might get different dates.

Why it works if you **write** with diesel is because it will always translate to UTC before writing it. I didnt change that behaviour (but we might want to).

Reply:
I think we can argue here that this is a bug fix. We should add it as potential breaking change to the changelog although.